include/rounding.h

```diff
@@ -113,11 +115,10 @@ void get_vpoly_center(Polytope &P) {
 
     boost::numeric::ublas::matrix<double> Ap(n,randPoints.size());
```

Comment:
possible TODO: remove dependence on ublas and copy to eigen
please add it as a comment